OWNERS

```diff
@@ -1,7 +1,6 @@
 # See the OWNERS docs at https://go.k8s.io/owners
 
 approvers:
-- alculquicondor
 - Huang-Wei
```

Comment:
@alculquicondor could you help also remove me from both approvers and reviewers? 🙏

Reply:
Done